lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoEnumConstantDeclarationCore.kt

```diff
@@ -63,6 +68,18 @@ internal class KoEnumConstantDeclarationCore private constructor(
         KoLocalDeclarationProviderCoreUtil.getKoLocalDeclarations(psiElements, this)
     }
 
+    override val arguments: List<KoArgumentDeclaration>
+        get() = ktEnumEntry
+            .initializerList
+            ?.initializers
+            ?.first()
+            ?.children
+            ?.filterIsInstance<KtValueArgumentList>()
+            ?.first()
```

Comment:
is there a scenario when `first` will be null and the entire code will fail? usually `firstOrNull` is safer